src/commands/logs.ts

```diff
@@ -23,6 +23,7 @@ interface IFile {
 
 export class ExportLogsCommand implements Command {
 	readonly id = 'gitpod.exportLogs';
+	static latestLSSHHost: string | undefined;
```

Comment:
@jeanp413  Will it make sense to persist last connection state in other place than memory? I tried locally:
- Is connected to workspace -> **has lssh logs**
- Window show disconnected and attempt to retry -> **not able to collect logs**
- Reload window -> **no last time's lssh logs**

I'm not sure where `in connection window` means other than the second point above. But it's not working

Reply:
it only applies when it's a normal window (not remote) and the user is trying to connect to a workspace and it fails before redirecting to the remote window

Reply:
> Window show disconnected and attempt to retry -> not able to collect logs

I thought this case is used to happen to users side as well, but as this PR has its own scope, let's publish it